test/functional/repository/aggregate-methods/repository-aggregate-methods.ts

```diff
@@ -3,8 +3,8 @@ import {
     closeTestingConnections,
     createTestingConnections,
 } from "../../../utils/test-utils"
-import { Repository } from "../../../../src/repository/Repository"
-import { DataSource } from "../../../../src/data-source/DataSource"
+import { Repository } from "../../../../src"
```

Comment:
remove redundant changes not related to your change. Just to make sure such changes won't bring any circular reference problems.